modules/bullet/rigid_body_bullet.cpp

```diff
@@ -343,23 +341,24 @@ void RigidBodyBullet::set_space(SpaceBullet *p_space) {
 		assert_no_constraints();
 
 		// Remove this object form the physics world
+		space->unregister_collision_object(this);
 		space->remove_rigid_body(this);
 	}
 
 	space = p_space;
 
 	if (space) {
-		space->add_rigid_body(this);
+		space->register_collision_object(this);
+		reload_body();
+		space->add_to_flush_queue(this);
 	}
 }
 
 void RigidBodyBullet::dispatch_callbacks() {
+	RigidCollisionObjectBullet::dispatch_callbacks();
```

Comment:
This is not needed. Check this: https://github.com/godotengine/godot/pull/40886#issuecomment-666964762

Reply:
So you want me to delete the call to `RigidCollisionObjectBullet::dispatch_callbacks()` inside of `RigidBodyBullet::dispatch_callbacks()`? I don't understand what that comment has to do with this line.